content/en/blog/_posts/2020-12-02-dockershim-faq.md

```diff
@@ -47,6 +47,15 @@ and other ecosystem groups to ensure a smooth transition and will evaluate thing
 as the situation evolves.
 
 
+### Can I still use dockershim after it is removed from Kubernetes?
+
+Update:
+[Mirantis and Docker have committed to maintaining the dockershim][mirantis] after
```

Comment:
[Mirantis and Docker have committed to maintaining the dockershim][mirantis] after

Reply:
Updated with requested change. Thanks!